src/test/drop/drop-erc721/lazyMint/lazyMint.tree

```diff
@@ -0,0 +1,33 @@
+function lazyMint(
+    uint256 _amount,
+    string calldata _baseURIForTokens,
+    bytes calldata _data
+)
+├── when called by a user without MINTER_ROLE
+│   └── it should revert
+└── when called by a user with MINTER_ROLE
+    ├── when _data.length == 0
+    │   ├── when _amount is equal to 0
+    │   │   └── it should revert
+    │   └── when _amount is greater than 0
+    │       ├── it should push batchId (_startId + _amountToMint) to the batchIds array
+    │       ├── it should set baseURI[batchId] as _baseURIForTokens
+    │       └── it should emit TokensLazyMinted with the parameters: startId, startId + amount - 1, _baseURIForTokens, _data
+    └── when _data.length > 0
+        ├── when _data invalid format
+        │   └── it should revert
+        └── when _data valid format
+            ├── it should decode _data into bytes memory encryptedURI and bytes32 provenanceHash
+            ├── if encryptedURI.length = 0
+            │   ├── it should push batchId (_startId + _amountToMint) to the batchIds array
+            │   ├── it should set baseURI[batchId] as _baseURIForTokens
+            │   └── it should emit TokensLazyMinted with the parameters: startId, startId + amount - 1, _baseURIForTokens, _data
+            ├── if provenanceHash = ""
+            │   ├── it should push batchId (_startId + _amountToMint) to the batchIds array
+            │   ├── it should set baseURI[batchId] as _baseURIForTokens
+            │   └── it should emit TokensLazyMinted with the parameters: startId, startId + amount - 1, _baseURIForTokens, _data
+            └── if encryptedURI.length > 0 and provenanceHash does not equal ""
```

Comment:
I don't think we use "if" syntax in any other trees? Just the `lazymint.tree` file might be outdated

Reply:
yup - must have missed updating this one.